### PR TITLE
Make func unit work with Qunit 2

### PR DESCRIPTION
**Pull request by shayaneumar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +61,7 @@
     "steal-jasmine": "0.0.3",
     "steal-mocha": "0.0.3",
     "steal-qunit": "^0.1.1",
+    "steal-qunit2": "git://github.com/shayaneumar/steal-qunit.git#78b22c0be77ea622a22f3385d9ebaced5f592e48",
```
Comment (shayaneumar) [inline package.json]: I have created a fork of steal_qunit repo and created a fork of that, named it to steal-qunit2.

This is done just for testing purposes, wanted to verify that code changes done for qunit 2 is working fine with both qunit 1x and qunit 2x.

**Comment by shayaneumar** [comment]
@matthewp need ur input on this.

**Comment by matthewp** [comment]
Looks good to me. Just need to update the qunit 2 package?

**Comment by shayaneumar** [comment]
@matthewp yes, just publish the steal_qunit2 which works with qunit2.

**Comment by shayaneumar** [comment]
@matthewp I waiting for package to be published so that I can proceed further.

**Comment by shayaneumar** [comment]
@matthewp, I am waiting on publishing.

**Comment by matthewp** [comment]
Which package are you waiting to be published?

**Comment by shayaneumar** [comment]
I have created a fork of steal–qunit which works with qunit 2 and gave it a name steal-qunit2. PR here works fine with that. I could not get my code changes working with steal-qunit1 which you earlier published. I am not sure what is the issue. can you publish a new package steal-qunit2 which works with qunit2?

**Comment by matthewp** [comment]
Sure.

**Comment by matthewp** [comment]
steal-qunit2 was published

File: package.json [hunk 2]
```diff
@@ -60,6 +61,7 @@
     "steal-jasmine": "0.0.3",
     "steal-mocha": "0.0.3",
     "steal-qunit": "^0.1.1",
+    "steal-qunit2": "1.0.0",
```
Comment (shayaneumar) [inline package.json]: @matthewp new package is used now.

**Comment by shayaneumar** [comment]
@matthewp  update PR with reference of new package steal_qunit2, please take a look.

**Comment by matthewp** [comment]
Cool. The title still says Don't Merge, is this mergable now? :D

**Comment by shayaneumar** [comment]
@matthewp title is changed we can merge it now. 😀

**Pull request merged**